Ports/readline/package.sh

```diff
@@ -7,3 +7,13 @@ config_sub_paths=("support/config.sub")
 use_fresh_config_sub=true
 files="https://ftpmirror.gnu.org/gnu/readline/readline-${version}.tar.gz readline-${version}.tar.gz 7589a2381a8419e68654a47623ce7dfcb756815c8fee726b98f90bf668af7bc6"
 auth_type=sha256
+configopts=(
+    "--disable-static"
+    "--enable-shared"
+)
+
+post_install() {
+    # readline specifies termcap as a dependency in its pkgconfig file, without checking if it exists.
+    # Remove it manually to keep other ports from discarding readline because termcap is supposedly missing.
+    sed -i '/^Requires.private:/s/termcap//' "${SERENITY_INSTALL_ROOT}/usr/local/lib/pkgconfig/readline.pc"
```

Comment:
Partially because I didn't know it existed, and partially (with hindsight, and in my opinion) because a `sed` command is much easier to understand than going through what is basically a `sed` alias that only we have.

Reply:
Can confirm this fails on mac
```
sed: 1: "/Users/erik/git/serenit ...": invalid command code e
```

Reply:
The `sed` invocation now works on both Linux and macOS.